include/irr/core/xxHash256.h

```diff
@@ -1,6 +1,6 @@
-// Copyright (C) 2019 DevSH Graphics Programming Sp. z O.O.
-// This file is part of the "IrrlichtBaW" Engine.
-// For conditions of distribution and use, see LICENSE.md
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
```

Comment:
xx256 hash comes from somewhere there should be a disclaimer for it.

Reply:
TODO

Reply:
It's already there
https://github.com/Przemog1/Nabla/blob/armageddon2/include/irr/core/xxHash256.h#L16